packages/teams-js/src/public/location.ts

```diff
@@ -46,82 +44,107 @@ export namespace location {
   }
 
   /**
-   * Fetches current user coordinates or allows user to choose location on map
-   *
-   * @param props {@link LocationProps} - Specifying how the location request is handled
-   * @returns Promise that will be fulfilled when the operation has completed
+   * Fetches current user coordinates
+   * @returns User's current location
    */
-  export function getLocation(props: LocationProps): Promise<Location>;
-  /**
-   * @deprecated
-   * As of 2.0.0, please use {@link location.getLocation location.getLocation(props: LocationProps): Promise\<Location\>} instead.
-   * @param props {@link LocationProps} - Specifying how the location request is handled
-   * @param callback - Callback to invoke when current user location is fetched
-   */
-  export function getLocation(props: LocationProps, callback: (error: SdkError, location: Location) => void): void;
```

Comment:
Is this supposed to be deleted? This is the backwards compatible function with a required callback?

Reply:
I see that it was actually moved down below, sorry about that